src/Management/test/CloudFoundryCore.Test/CloudFoundryHostBuilderExtensionsTest.cs

```diff
@@ -124,11 +126,28 @@ public async Task AddCloudFoundryActuators_IHostBuilder_IStartupFilterFires()
         {
             // Arrange
             var hostBuilder = new HostBuilder()
-                .ConfigureWebHost(c => c.UseTestServer().Configure(app => { }))
+                .ConfigureWebHost(testServerWithRouting)
                 .ConfigureAppConfiguration(cbuilder => cbuilder.AddInMemoryCollection(managementSettings));
 
             // Act
-            var host = await hostBuilder.AddCloudFoundryActuators(MediaTypeVersion.V1, ActuatorContext.ActuatorAndCloudFoundry).StartAsync();
+            var host = await hostBuilder.AddCloudFoundryActuators(MediaTypeVersion.V2).StartAsync();
+
+            // Assert general success...
+            //   not sure how to actually validate the StartupFilter worked,
```

Comment:
Check [via HTTP](https://docs.microsoft.com/en-us/aspnet/core/test/middleware?view=aspnetcore-3.1)

Reply:
This was your comment I believe - from prior versions - should I remove the comment ? we have other places where we make a call?

Reply:
hah, that figures... we should either remove the comments or make the http calls. I know there's more than one instance of this comment but I don't know offhand if each startupfilter has a test that calls.... I think there should be at least one http call for each